j2735_msgs/msg/Point.msg

```diff
@@ -0,0 +1,29 @@
+#
+# Point.msg
+#
+# J2735 2016 message format.
+#
+# @author Misheel Bayartsengel
+# @version 0.1
+#
+# Description
+# ...
+
+# Point ::= IMPLICIT SEQUENCE {
+# 	x INTEGER, -- units are cm for all of these
+# 	y INTEGER,
+# 	z INTEGER OPTIONAL,
+# 	width INTEGER
+# }
+
+# X ::= INTEGER (−2147483648, 2147483647)
+#   -- units are in cm
+int32 x
+int32 y
+int32 z
+
+bool z_exists
+
+# Width ::= INTEGER (0, 4294967295)
+#   -- units are in cm
+uint32 width
```

Comment:
I think uint16 would be sufficient here. I believe this width is the width of the lane. Uint16 gives a half mile wide lane.